src/validator.cc

```diff
@@ -959,6 +1023,42 @@ Result Validator::CheckGet(const Action* action, Type* out_type) {
   return Result::Ok;
 }
 
+Result Validator::CheckExceptVar(const Var* var, const Exception** out_except) {
+  Index index;
+  if (WABT_FAILED(
+          CheckVar(current_module_->excepts.size(), var, "except", &index))) {
+    return Result::Error;
+  }
+  if (out_except)
+    *out_except = current_module_->excepts[index];
+  return Result::Ok;
+}
+
+void Validator::CheckExcept(const Location* loc, const Exception* except) {
+  for (Type ty : except->sig) {
+    switch (ty) {
+      default:
+        PrintError(loc, "Invalid exception type: %s", get_type_name(ty));
+        break;
```

Comment:
I'm used to seeing default that the end of the switch.  Is this a style used elsewhere in wabt?